mjulialobo/decor-catalog
# feat(axios): api connection

## Why
Create product card and receive values from API+ a few fixes;

## What was done
- Added Axios;
- Made API Call;
- Created ProductCard component + tests;
- Adjusted some Theme details
- Changed folders structure

# How to test
- Run the project and open the home, you should see the product cards with API information;

## Result
![image](https://github.com/mjulialobo/decor-catalog/assets/65983895/d453ee0e-a55b-4c89-8390-437bdcb80faf)
